install/ansible/cri-containerd.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- hosts: all 
+  become: true
+  # Setting the environment variables that might be needed for the entire installation
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/opt/golang/bin:/usr/local/go/bin:/usr/local/bin"
+    GOPATH: "/opt/golang"
+    GOBIN: "/opt/golang/bin"
+  tasks:
+    - include_vars: vars/vars.yaml # Contains tasks variables for installer
+    - include: tasks/bootstrap_ubuntu.yaml #Contains tasks bootstrap components for ubuntu systems 
+      when: ansible_distribution == "Ubuntu"
+    - include: tasks/bootstrap_centos.yaml #Contains tasks bootstrap components for centos systems
+      when: ansible_distribution == "CentOS"
+    - include: tasks/k8.yaml #Contains tasks kubernetes component installation
+    - include: tasks/build.yaml #Contains tasks for pulling containerd and cri-containerd components
+       
+    - name: "Start Containerd"
+      systemd: name=containerd daemon_reload=yes state=started enabled=yes
+    
+    - name: "Start CRI-Containerd"
+      systemd: name=cri-containerd daemon_reload=yes state=started enabled=yes
+    
+    - name: "Set bridge-nf-call-iptables"
+      lineinfile:
+        line: "net/bridge/bridge-nf-call-iptables = 1"
+        dest: /etc/sysctl.conf
+        insertafter: 'EOF'
+        regexp: '\/net\/bridge\/bridge-nf-call-iptables = 1'
+        state: present
+      ignore_errors: true
+    
+    - name: "Check kubelet args in kubelet config"
```

Comment:
Can we do something similar with cri-o? Just drop in another file https://github.com/cri-o/cri-o-ansible/blob/master/cri-o.yml#L217-L220.

Reply:
Hmm what about cni configs and stuff ? Does kubelet read from all the config filed in the folder ?

Reply:
> Hmm what about cni configs and stuff?

What do you mean?

> Does kubelet read from all the config filed in the folder ?

I guess that's how systemd works. Or cri-o installation won't work then. :)

Reply:
@Random-Liu  I dont see any harm in keeping this way. I am not sure how that works with multiple config files. This seems to be easier if someone peeps into config file for kubelet its all in one place. Unless there is any disadvantage with this approach. Here I have ensured we dont double insert the config as well so if we run the ansible playbook multiple times also it will not insert the configs